app/models/scope_limitation.rb

```diff
@@ -0,0 +1,7 @@
+class ScopeLimitation < ApplicationRecord
```

Comment:
not a blocker - would we normally create a new model spec for a new model? Possibly not worth it for this though (at least not yet).

Reply:
It was on oversight on my part - thanks for spotting :+1:
I have added a very basic placeholder and we can expand as needed